src/clients/ntp-client_proposal.rb

```diff
@@ -315,10 +300,13 @@ def AskUser
     def WriteNtpSettings(ntp_servers, ntp_server, run_service)
```

Comment:
this function has interesting parameters which deserve some documentation. E.g. if i understand it correctly ```ntp_server``` is ignored when ```ntp_servers``` is not empty.

Reply:
yes, old code, but will document it.